packages/story-editor/src/components/checklist/constants.js

```diff
@@ -93,6 +93,13 @@ export const ACCESSIBILITY_COPY = {
       'web-stories'
     ),
   },
+  linkTappableRegionTooBig: {
+    title: __('Reduce link size', 'web-stories'),
+    footer: __(
+      'Links cannot cover a large majority of the page. Use smaller elements for links.',
```

Comment:
💭 this copy feels a little confusing to me. Maybe something more along the lines of

`Elements with links must not cover over 80% of the page. Decrease the size of elements with links.`

Reply:
@choumx  ^^ thoughts on copy update?

Reply:
Not sure we should use "80%", how about:

> Elements with links must not cover most of the page. Decrease the size of elements with links.

Reply:
Sure! new ticket here: https://github.com/google/web-stories-wp/issues/9712